src/client/game/map_info.tsx

```diff
@@ -22,25 +12,13 @@ export function MapInfo({ gameKey }: { gameKey: string }) {
       <li>Supports: {selectedMap.minPlayers}{selectedMap.minPlayers === selectedMap.maxPlayers ? '' : `-${selectedMap.maxPlayers}`} players</li>
     </ul>
     <h3>Rules</h3>
-    <AlternateRules gameKey={gameKey} />
+    <AlternateRules selectedMap={selectedMap} />
   </div>
 }
 
-export function AlternateRules({ gameKey }: { gameKey: string }) {
-  switch (gameKey) {
-    case 'ireland':
-      return <IrelandRules />;
-    case SwedenRecyclingMapSettings.key:
-      return <SwedenRecyclingRules />;
-    case CyprusMapSettings.key:
-      return <CyprusRules />;
-    case MadagascarMapSettings.key:
-      return <MadagascarRules />;
-    case IndiaMapSettings.key:
-      return <IndiaRules />;
-    case KoreaMapSettings.key:
-      return <KoreaRules />;
-    default:
-      return <p>No changes from base game.</p>;
+export function AlternateRules({ selectedMap }: { selectedMap: MapSettings }) {
+  if (selectedMap.getMapRules) {
```

Comment:
Maybe getMapRules should be a required property. I'm not aware of any maps that use the base rules.